CefSharp.Core/ManagedCefBrowserAdapter.cpp

```diff
@@ -61,6 +61,23 @@ void ManagedCefBrowserAdapter::CreateBrowser(IWindowInfo^ windowInfo, BrowserSet
 
     extraInfo->SetBool("LegacyBindingEnabled", legacyBindingEnabled);
 
+    // Retrieve the configurable binding property names, throw exception if empty or illegal characters
+    auto jsBindingPropertyName = CefSharpSettings::JavascriptBindingPropertyName;
+    auto jsBindingPropertyNameCamelCase = CefSharpSettings::JavascriptBindingPropertyNameCamelCase;
+    if (!StringCheck::EnsureLettersAndNumbers(jsBindingPropertyName) || !StringCheck::EnsureLettersAndNumbers(jsBindingPropertyNameCamelCase))
+    {
+        throw gcnew InvalidOperationException("CefBrowserHost::CreateBrowser invalid or illegal characters used for binding property names. Alphanumeric and underscores characters only.");
+    }
+
+    extraInfo->SetString("JsBindingPropertyName", StringUtils::ToNative(jsBindingPropertyName));
+    extraInfo->SetString("JsBindingPropertyNameCamelCase", StringUtils::ToNative(jsBindingPropertyNameCamelCase));
+
+    if (!CefBrowserHost::CreateBrowser(*cefWindowInfoWrapper->GetWindowInfo(), _clientAdapter.get(), addressNative,
```

Comment:
Unit tests are failing as CreateBrowser is incorrectly being called twice.

Reply:
Oops! Ok I've removed the duplicate method.